Week3/outdated.py

```diff
@@ -0,0 +1,38 @@
+monlist = [
+    "January",
+    "February",
+    "March",
+    "April",
+    "May",
+    "June",
+    "July",
+    "August",
+    "September",
+    "October",
+    "November",
+    "December"
+]
+
+while True:
+    date = input("Date: ")
+    #9/8/1636 
+    try:
+        month, day, year = date.split("/", 2)
+        if (1<= int(month) <= 31)and (1 <= int(month) <= 12):
+            print(f"{year}-{int(month):02}-{int(day):02}")
+        break
+    
```

Comment:
This is what I think the above code should be implemented, think about it and we can discuss
```
if (1<= int(month) <= 31)and (1 <= int(month) <= 12):
        print(f"{year}-{int(month):02}-{int(day):02}")
        break
else:
        raise ValueError()
```